development/tools/pkg/rendertemplates/rendertemplates.go

```diff
@@ -196,8 +196,8 @@ func (ft FromTo) String() string {
 func (tplCfg *TemplateConfig) mergeConfigs(config *Config) {
 	for _, render := range tplCfg.Render {
 		render.mergeConfigs(config.GlobalSets)
-		// check if there are any component jobs in merged config and generate config for such jobs for each supported release
-		render.GenerateComponentJobs(config.Global)
+		// generate component jobs and append all jobs to the list for rendering
```

Comment:
This function now is donig much more than merging configs. It merges configs, generates component jobs, and add them to them somewhere. This should be refactored in other way.